spec.html

```diff
@@ -4417,7 +4417,7 @@ <h1>
         <dl class="header">
         </dl>
         <emu-alg>
-          1. If Type(_Obj_) is not Object, throw a *TypeError* exception.
+          1. If _Obj_ is not an Object, throw a *TypeError* exception.
```

Comment:
i kind of think that this reads much less clearly for Object comparisons - with Type, i can look and confirm whether functions are Objects or not, but with this definition i'm left wondering.

Reply:
To clarify, are you saying this?:
- In the rendered spec, the use of "Type" here is linked to section 6 "ECMAScript Data Types and Values" which (eventually) answers the question of whether a function is an Object.
- Whereas in the rendered spec, "is not an Object" doesn't link to anything.

Reply:
Yes, exactly.

Reply:
In that case, we could make "is not an Object" link to something. E.g., somewhere add a sentence with `<dfn variants="is not an Object">is an Object</dfn>`. In fact, this would be an improvement over the status quo, because the fact that a function is an object could be stated right there at the `<dfn>`, whereas currently that information is rather a long way from the definition of `Type()`.

Reply:
Added `<dfn>`s. I think it's actually pretty nice.